src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java

```diff
@@ -383,7 +385,14 @@ private void readNextRecord() throws IOException {
                     e.setSource(path);
                     throw e;
                 } catch (NumberFormatException e) {
-                    final String error = "Error parsing line at byte position: " + source;
+
+                    final String error;
+                    if (previousRecord == null) {
+                        error = String.format("Error parsing %s at the first record", source);
+                    } else {
+                        error = String.format("Error parsing %s just after record at: %s:%d-%d",
```

Comment:
It's not clear to me that source.toString() will be meaningful at all.  I might just drop that.  Path is useful but that should be included already.